src/components/PopupWithForm.js

```diff
@@ -15,19 +14,47 @@ export default class PopupWithForm extends Popup {
     });
     return inputValues;
   }
-  // close() {
-  //   this._popupForm.reset();
-  //   super.close();
-  // }
+
+  open() {
+    this._popupForm.reset();
```

Comment:
You can make method `resetForm` here


```js
  resetForm() {
    this._popupForm.reset();
  }


```

And you can call it in `then` after submitting